src/property-accessors.coffee

```diff
@@ -1,5 +1,5 @@
 Mixin = require 'mixto'
-WeakMap = global.WeakMap ? require('harmony-collections').WeakMap
+WeakMap = require 'es6-weak-map'
```

Comment:
Okay, I'm going to add that back here then so that the library only needs to be required as a fallback.

We try to defer optional requires to minimize the initial require time since. This library is loaded at startup of Atom.